### PR TITLE
Update DE-Finanzamtmeldung.md

### PR DESCRIPTION
**Pull request by aedler-ft**

### CODE REVIEW

**Review by AxelKutschera — APPROVED** [review]
Some suggestions.

File: qna/DE-Finanzamtmeldung.md [hunk 1]
```diff
@@ -18,6 +18,19 @@ lang-de, market-de, middleware, PosCreator, PosDealer, Consultant
 
 Laut [Nichtbeanstandungsregelung](https://docs.fiskaltrust.cloud/doc/productdescription-de-doc/product-service-description/DE-rechtliche-grundlagen.html#aktuell-gültige-rechtliche-grundlagen) ist von der Mitteilung (Meldung der TSE an das Finanzamt, Anm.) nach § 146a Absatz 4 AO bis zum Einsatz einer elektronischen Übermittlungsmöglichkeit abzusehen. Der Zeitpunkt des Einsatzes der elektronischen Übermittlungsmöglichkeit wird im Bundessteuerblatt Teil I gesondert bekannt gegeben.
 
-Es sind aktuell daher weder der Zeitpunkt noch der Inhalt der zu übermittelnden Informationen bis zur oben beschriebenen Veröffentlichung der elektronischen Übermittlungsmöglichkeit bekannt. 
+Es ist aktuell daher der Zeitpunkt der zu übermittelnden Informationen bis zur oben beschriebenen Veröffentlichung der elektronischen Übermittlungsmöglichkeit nicht bekannt. 
+
+Wer aufzeichnungspflichtige Geschäftsvorfälle oder andere Vorgänge mit Hilfe eines elektronischen Aufzeichnungssystems, hat dem zuständigen Finanzamt nach amtlich vorgeschriebenen Vordruck mitzuteilen:
```
[the remark this answers is not in the document]
Comment (aedler-ft) [reply qna/DE-Finanzamtmeldung.md:23]: ok

File: qna/DE-Finanzamtmeldung.md [hunk 2]
```diff
@@ -18,6 +18,19 @@ lang-de, market-de, middleware, PosCreator, PosDealer, Consultant
...
+1. Name des Steuerpflichtigen, 
+2. Steuernummer des Steuerpflichtigen, 
+3. Art der zertifizierten technischen Sicherheitseinrichtung, 
+4. Art des verwendeten elektronischen Aufzeichnungssystems, 
+5. Anzahl der verwendeten elektronischen Aufzeichnungssysteme, 
+6. Seriennummer des verwendeten elektronischen Aufzeichnungssystems, 
+7. Datum der Anschaffung des verwendeten elektronischen Aufzeichnungssystems, 
+8. Datum der Außerbetriebnahme des verwendeten elektronischen Aufzeichnungssystems. 
+
+Die Mitteilung nach Satz 1 ist innerhalb eines Monats nach Anschaffung oder Außerbetriebnahme des elektronischen Aufzeichnungssystems zu erstatten.
```
[the remark this answers is not in the document]
Comment (aedler-ft) [reply qna/DE-Finanzamtmeldung.md:34]: ok

File: qna/DE-Finanzamtmeldung.md [hunk 3]
```diff
@@ -18,6 +18,19 @@ lang-de, market-de, middleware, PosCreator, PosDealer, Consultant
 
 Laut [Nichtbeanstandungsregelung](https://docs.fiskaltrust.cloud/doc/productdescription-de-doc/product-service-description/DE-rechtliche-grundlagen.html#aktuell-gültige-rechtliche-grundlagen) ist von der Mitteilung (Meldung der TSE an das Finanzamt, Anm.) nach § 146a Absatz 4 AO bis zum Einsatz einer elektronischen Übermittlungsmöglichkeit abzusehen. Der Zeitpunkt des Einsatzes der elektronischen Übermittlungsmöglichkeit wird im Bundessteuerblatt Teil I gesondert bekannt gegeben.
 
-Es sind aktuell daher weder der Zeitpunkt noch der Inhalt der zu übermittelnden Informationen bis zur oben beschriebenen Veröffentlichung der elektronischen Übermittlungsmöglichkeit bekannt. 
+Es ist aktuell daher der Zeitpunkt der zu übermittelnden Informationen bis zur oben beschriebenen Veröffentlichung der elektronischen Übermittlungsmöglichkeit nicht bekannt.
```
[the remark this answers is not in the document]
Comment (aedler-ft) [reply qna/DE-Finanzamtmeldung.md:21]: ok

**Pull request merged**